converter/src/main/java/gov/cms/qpp/conversion/decode/QrdaDecoderEngine.java

```diff
@@ -144,11 +146,13 @@ private DecodeData decodeChildren(final Element element, final Node parentNode)
 
 		List<Element> childElements = element.getChildren();
 
+		List<Element> filteredChildElements = getUniqueTemplateIdElements(childElements);
```

Comment:
Might as well not bother creating `childElements` since it's not used anywhere else.